services/actions/notifier.go

```diff
@@ -526,3 +526,38 @@ func (n *actionsNotifier) NotifyPullRequestChangeTargetBranch(ctx context.Contex
 		WithPullRequest(pr).
 		Notify(ctx)
 }
+
+func (n *actionsNotifier) NotifyNewWikiPage(ctx context.Context, doer *user_model.User, repo *repo_model.Repository, page, comment string) {
+	ctx = withMethod(ctx, "NotifyNewWikiPage")
+
+	newNotifyInput(repo, doer, webhook_module.HookEventWiki).WithPayload(&api.WikiPayload{
+		Action:     api.HookWikiCreated,
+		Repository: convert.ToRepo(ctx, repo, perm_model.AccessModeOwner),
```

Comment:
Why not `AccessLevel(ctx, doer, repo)`?

Reply:
The access mode is `AccessModeOwner` in `webhookNotifier`, so I also use `AccessModeOwner` here.
https://github.com/go-gitea/gitea/blob/1c8bc4081a4f4d0d921ac218cb724ce97924d410/services/webhook/notifier.go#L468

Reply:
Just a question.

webhook payload:
![image](https://user-images.githubusercontent.com/18380374/231967557-89158ec6-8528-40b8-b11a-b56c9d4a41b1.png)
code:
![image](https://user-images.githubusercontent.com/18380374/231967597-0f757458-69f2-4ae9-a4b7-9eceaedebb24.png)

It seems that `repository.permissions` in webhook payload is related to the permission here.
I can't understand the meaning of  `repository.permissions` here ( who's permision? ) . And can't find the definition in [document](https://docs.gitea.io/en-us/webhooks/)

Reply:
I think repository permission should equal to code unit permission except admin.

Reply:
But if we can have a more clear definition, that's better.